[OpenTelemetry Instrumentation] Integration test libraries failing in nightly runs

OpenTelemetry Instrumentation nightly test runs are failing with:

> Trace: [Internal Error] 
>     at D:\a\_work\1\s\common\tools\dev-tool\src\index.ts:10:11
>     at processTicksAndRejections (node:internal/process/task_queues:95:5)
> Error: Failed calling dev-tool run test:node-js-input --no-test-proxy=true.  Exit code: 255
> ##[error]Cmd.exe exited with code '255'.

For more details check here:
[https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3388447&view=results](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3388447&view=results)

--- a/sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/package.json
+++ b/sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/package.json
@@ -29,7 +29,7 @@
     "generate:client": "autorest --typescript ./swagger/README.md",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "integration-test:browser": "echo skipped",
-    "integration-test:node": "echo skipped",
+    "integration-test:node": "dev-tool run test:vitest --no-test-proxy",
     "lint": "eslint package.json api-extractor.json README.md src test",
     "lint:fix": "eslint package.json api-extractor.json README.md src test --fix --fix-type [problem,suggestion]",
     "pack": "npm pack 2>&1",

